src/appkit.cs

```diff
@@ -760,6 +760,29 @@ interface NSApplication : NSAccessibilityElementProtocol, NSAccessibility {
 		void EnumerateWindows (NSWindowListOptions options, NSApplicationEnumerateWindowsHandler block);
 	}
 
+	[Static]
+	interface NSAboutPanelOption {
```

Comment:
See my comment above when Sebastien asked about it. 

https://github.com/xamarin/xamarin-macios/pull/2245#discussion_r123825524